packages/design-system/src/components/Tabs/Tabs.tsx

```diff
@@ -31,6 +35,10 @@ export interface TabsProps {
    * Additional classes to be added to the component wrapping the tabs
    */
   tablistClassName?: string;
+  /**
+   * Helpful description of the tabs for screenreaders. An alias for `ariaLabel` specifically added to improve accessibility for the web component version of this component.
+   */
+  tabsAriaLabel?: string;
```

Comment:
Is this prop meant only for the web component?

Reply:
Yes, as far as I'm aware, use in the web component is its only intended use case for now, and I can't think of other contexts where it would apply. Are there other possible scenarios where it might come in handy?

Reply:
In that case, should it be removed from this file?

Reply:
Yeah @tamara-corbalt this should get removed, since the attribute value on the web component is being added to the prop value [here](https://github.com/CMSgov/design-system/pull/3264/files#diff-01521597bdd1f8032256b3e0f2d9e6c48812f08b37906c7f321beaf507013cd8R26)

Reply:
@pwolfert, @jack-ryan-nava-pbc, thanks for the call out :) Removed it